cl/search/models.py

```diff
@@ -3472,6 +3472,7 @@ class Meta:
                 name="unique_opinion_ordering_key",
             )
         ]
+        ordering = ["ordering_key"]
```

Comment:
I don't think I'd approach this this way. The problem with doing this is that it would apply the ordering everywhere, which imposes a performance penalty to queries where we don't care about the ordering. I think this is one of those django features that are considered an anti-pattern these days. 

Can you refactor to apply the ordering only where we need and want it?